src/lib/support/SafeInt.h

```diff
@@ -37,11 +37,10 @@ namespace chip {
 template <typename T, typename U, std::enable_if_t<std::is_integral<T>::value, int> = 0>
 bool CanCastTo(U arg)
 {
-    using namespace std;
```

Comment:
Probably OK, but this use was safe: it's scoped to this function, and this function does not use `to_underlying`.

Reply:
Restored it.